cli/popper/gha.py

```diff
@@ -182,22 +183,22 @@ def instantiate_runners(self):
                 a['runner'] = HostRunner(a, self.workspace,
                                          self.env, self.timeout)
 
-    def run(self, action_name=None, reuse=False):
+    def run(self, action_name=None, reuse=False, verbose=False):
         """Run the pipeline or a specific action"""
         os.environ['WORKSPACE'] = self.workspace
 
         self.download_actions()
         self.instantiate_runners()
```

Comment:
as a way of minimizing the number of functions that get the `verbose` parameter, we can pass it to the `instantiate_runners` method and then get it on the constructor of each runner, and make a `verbose` member of the base `AcitonRunner` class.